### PR TITLE
chore: minor changes to existing features

### PR DESCRIPTION
**Pull request by gbartolini**

### CODE REVIEW

File: postgres/spec/feature_matrix.yaml [hunk 1]
```diff
@@ -627,7 +628,7 @@ categories:
         type: boolean
         description: |
           The operator provides a mechanism to expose all the logs of the managed Postgres instances to a centralized logging tool.
-          The logs must be decorated with extra metadata that includes the Pod name and namespace, the cluster name, the role of the Postgres instance (e.g. primary, replica, standby-leader, etc.) and the timestamp that will be available to be used to filter logs entries.
+          The logs must be decorated with extra metadata in order to provide semantic meaning, including the Pod name and namespace, the cluster name, the role of the Postgres instance (e.g. primary, replica, standby-leader, etc.) and the timestamp that will be available to be used to filter logs entries.
```
Comment (jsilvela) [inline postgres/spec/feature_matrix.yaml:631]:
```suggestion
          The logs must be decorated with extra metadata in order to provide context and meaning, including the Pod name and namespace, the cluster name, the role of the Postgres instance (e.g. primary, replica, standby-leader, etc.) and the timestamp that will be available to be used to filter logs entries.
```

File: postgres/spec/feature_matrix.md [hunk 2]
```diff
@@ -112,11 +112,11 @@
...
-| exdel | Exposed decorated logs | boolean | The operator provides a mechanism to expose all the logs of the managed Postgres instances to a centralized logging tool.<br />The logs must be decorated with extra metadata that includes the Pod name and namespace, the cluster name, the role of the Postgres instance (e.g. primary, replica, standby-leader, etc.) and the timestamp that will be available to be used to filter logs entries.<br />There is no need to configure the tool in order to obtain required extra metadata from the logs. |  |  |
+| exdel | Exposed decorated logs | boolean | The operator provides a mechanism to expose all the logs of the managed Postgres instances to a centralized logging tool.<br />The logs must be decorated with extra metadata in order to provide semantic meaning, including the Pod name and namespace, the cluster name, the role of the Postgres instance (e.g. primary, replica, standby-leader, etc.) and the timestamp that will be available to be used to filter logs entries.<br />There is no need to configure the tool in order to obtain required extra metadata from the logs. |  |  |
```
Comment (jsilvela) [inline postgres/spec/feature_matrix.md:119]:
```suggestion
| exdel | Exposed decorated logs | boolean | The operator provides a mechanism to expose all the logs of the managed Postgres instances to a centralized logging tool.<br />The logs must be decorated with extra metadata in order to provide context and meaning, including the Pod name and namespace, the cluster name, the role of the Postgres instance (e.g. primary, replica, standby-leader, etc.) and the timestamp that will be available to be used to filter logs entries.<br />There is no need to configure the tool in order to obtain required extra metadata from the logs. |  |  |
```

**Review by jsilvela — COMMENTED** [review]
I don't have permission to push to your repo, so leaving a couple of comments.
There is a typo, both in the MD and the YAML, on feature `serep`: look for "remainding".

**Comment by ahachete** [comment]
Hi @jsilvela Thank you very much for your comments. Since you couldn't push to @gbartolini branch, I opted to merge this one already and, if you could fork now the repo and submit your own patch with these changes, I'd really appreciate it.

**Pull request merged**